testing/munin/munin-node.initd

```diff
@@ -0,0 +1,22 @@
+#!/sbin/openrc-run
+
+# munin-node init.d file for alpine linux.
+
+name=munin-node
+
+pidfile=/run/munin/${name}.pid
+
+command=/usr/sbin/munin-node
+command_args="${EXTRA_OPTS}"
+command_background="no"
```

Comment:
Actually yes, it works as expected (tested on one of my servers). With command_background=yes the the status of the process is „crashed“ because the munin-node puts itself into background and there is no way to prevent it from doing so.
As munin-node is creating its own pid-file this should be ok, right?